In [1]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt

In [2]:
datahw01 = pd.read_csv('datasethw01.csv')

In [3]:
datahw01.size, datahw01.shape

(8370, (930, 9))

In [57]:
for i in range(9):
    print(i+1, datahw01.iloc[i+1].nonzero()[0].size)


1 9
2 9
3 8
4 9
5 8
6 9
7 8
8 9
9 9


/home/yuri/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:2: FutureWarning: Series.nonzero() is deprecated and will be removed in a future version.Use Series.to_numpy().nonzero() instead
  


In [16]:
datahw01['Call Test Result'].nunique(), datahw01['Call Test Technology'].nunique()

(2, 3)

In [48]:
grouped = datahw01.groupby('Call Test Result')

for name, group in grouped:
    print(group.shape)
    
grouped = datahw01.groupby('Call Test Technology')
print('Tecnologias:')
for name, group in grouped:
    print(name)
    print(group.shape)

(11, 9)
(919, 9)
Tecnologias:
0
(21, 9)
1
(270, 9)
2
(639, 9)



 - Dados: MOS

sucesso no teste = 919  
falhas no teste  = 11

- Tecnologias:

LTE = 639
UMTS = 270
GSM = 21

- 